services/storage/src/simcore_service_storage/handlers_files.py

```diff
@@ -287,6 +290,9 @@ async def complete_upload_file(request: web.Request) -> web.Response:
         )
         .with_query(user_id=query_params.user_id)
     )
+    complete_task_state_url = AnyHttpUrl(
+        url=f"http://{ip_addr}:{port}{route}", scheme="http"
+    )
```

Comment:
question, is it needed to hard-code the scheme here?
in case we ever go https inside the stack this will be difficult to find out.
Also afterwards it is parsed again as a ```AnyUrl``` which does not need a scheme in front.

Reply:
I have changed to `AnyUrl` but it seems that one also needs a scheme (it was also present in the previous code). However, now I get the scheme directly from `request.url` so that should be more maintainable.